src/OperatorMsg.chpl

```diff
@@ -1346,6 +1346,11 @@ module OperatorMsg
                     when "|=" {l.a |= r.a;}
                     when "&=" {l.a &= r.a;}
                     when "^=" {l.a ^= r.a;}
+                    when "+=" {
+                      ref la = l.a;
+                      ref ra = r.a;
+                      [(li,ri) in zip(la,ra)] li = if (li == true || ri == true) then true else false;
+                    }
```

Comment:
I might be missing something, but this seems equivalent to the `l.a |= r.a;` case

Reply:
You are totally right, I totally overlooked the operation right above where I added this.